support/cas-server-support-pac4j-core-clients/src/main/java/org/apereo/cas/support/pac4j/web/flow/DelegatedClientAuthenticationAction.java

```diff
@@ -93,18 +96,28 @@
     private final String themeParamName;
     private final String localParamName;
     private final boolean autoRedirect;
+    private final Store<String, CommonProfile> profileStore;
     private final ServicesManager servicesManager;
 
     public DelegatedClientAuthenticationAction(final Clients clients, final AuthenticationSystemSupport authenticationSystemSupport,
                                                final CentralAuthenticationService centralAuthenticationService, final String themeParamName,
-                                               final String localParamName, final boolean autoRedirect, final ServicesManager servicesManager) {
+                                               final String localParamName, final boolean autoRedirect, final ServicesManager servicesManager,
+                                               final Store<String, CommonProfile> profileStore) {
+        Assert.notNull(clients, "The clients cannot be null.");
```

Comment:
Remove all. Move to the place where the action is created and potentially use NotNull annotations.

Reply:
The whole constructor has gone with the introduction of Lombok.